### DISCUSSION TITLE
Github Secrets Tied to User Account

### DISCUSSION THREAD

**Discussion by dvargasfuertes** [community/community#27674] [Apps, API and Webhooks] [answered]
I was wondering if there was a way to tied a Github Secret to my account itself, so that they would be available to any GitHub repository that uses it. One example use case of this is an NPM API Token. Instead of adding the same token to each of my repos hosting an NPM package for ease of publishing, I would like to add a single one to my account that all repositories inherit from. Is this possible? I saw that this is possible for orgs, but couldn't find anything on user accounts.

**Comment by jsoref** [comment, marked as answer]
Create an org, put your repos in the org, create an org secret.

There's no requirement that you have other members in your org. Orgs can be free. There is no limit on orgs.

My feeling is that it's a design choice. The default process is that a user will fork a random repository into their user account. It'd be *really* bad if that simple action enabled a random repository to steal that user's secrets.

**Reply by dvargasfuertes** [reply]
Not sure I agree that stealing a user's secret would be enabled that much further in this scenario, but I understand the sentiment. It appears there isn't a feature I was unaware of and I should proceed with creating an org. Thank you!
